### PR TITLE
Fat 2760 c357552

### PR DESCRIPTION
**Pull request by TetianaParanich**
***
https://issues.folio.org/browse/FAT-2760
***
https://foliotest.testrail.io/index.php?/cases/view/357552
***

### CODE REVIEW

File: cypress/support/fragments/data-export/exportMappingProfile/exportFieldMappingProfiles.js [hunk 1]
```diff
@@ -5,9 +5,21 @@ const openNewMappingProfileForm = () => {
   cy.do(Button('New').click());
 };
 
+const saveMappingProfile = () => {
+  cy.do(Button('Save & close').click());
+  cy.expect(Button('Save & close').absent());
```
Comment (IhorBohdan) [inline cypress/support/fragments/data-export/exportMappingProfile/exportFieldMappingProfiles.js:10]: Move to const

File: cypress/support/fragments/inventory/inventorySearchAndFilter.js [hunk 2]
```diff
@@ -451,4 +451,12 @@ export default {
   selectViewHoldings() {
     cy.do(viewHoldingButton.click());
   },
+
+  filterItemByStatisticalCode:(code) => {
+    cy.do([
+      Button({ id:'accordion-toggle-button-itemsStatisticalCodeIds' }).click(),
+      Accordion({ id:'itemsStatisticalCodeIds' }).find(TextField()).fillIn(code),
+    ]);
+    cy.do(Accordion({ id:'itemsStatisticalCodeIds' }).find(Checkbox(code)).click());
```
Comment (IhorBohdan) [inline cypress/support/fragments/inventory/inventorySearchAndFilter.js:460]: move to const

**Review by IhorBohdan — APPROVED** [review]

**Review by MaksimTerekhov — APPROVED** [review]

File: cypress/support/fragments/data-export/exportMappingProfile/exportFieldMappingProfiles.js (hunk 1 above)
Comment (TetianaParanich) [reply cypress/support/fragments/data-export/exportMappingProfile/exportFieldMappingProfiles.js]: Done

File: cypress/support/fragments/inventory/inventorySearchAndFilter.js (hunk 2 above)
Comment (TetianaParanich) [reply cypress/support/fragments/inventory/inventorySearchAndFilter.js]: Done

File: cypress/support/fragments/data_import/mapping_profiles/newFieldMappingProfile.js [hunk 3]
```diff
@@ -242,7 +242,7 @@ export default {
 
     cy.do(Select({ name: statisticalCodeFieldName }).choose(actions.addTheseToExisting));
     cy.do(Button('Add statistical code').click());
-    cy.do(TextField('Statistical code').fillIn(name));
+    cy.do(TextField('Statistical code').fillIn(`"${name}"`));
```
Comment (MaksimTerekhov) [inline cypress/support/fragments/data_import/mapping_profiles/newFieldMappingProfile.js:245]: you can use just

`cy.do(TextField('Statistical code').fillIn(name));`

Comment (TetianaParanich) [reply cypress/support/fragments/data_import/mapping_profiles/newFieldMappingProfile.js:245]: I need double quotes when I'm creating mapping profile, but I don't need them when I'm searching item in Inventory by statistical code. That's why I use this constriction in creating function

**Pull request merged**